MediaBrowser.MediaEncoding/Probing/ProbeResultNormalizer.cs

```diff
@@ -1531,24 +1449,27 @@ private void FetchWtvInfo(MediaInfo video, InternalMediaInfoResult data)
 
         private void ExtractTimestamp(MediaInfo video)
         {
-            if (video.VideoType == VideoType.VideoFile)
+            if (video.VideoType != VideoType.VideoFile)
             {
-                if (string.Equals(video.Container, "mpeg2ts", StringComparison.OrdinalIgnoreCase) ||
-                    string.Equals(video.Container, "m2ts", StringComparison.OrdinalIgnoreCase) ||
-                    string.Equals(video.Container, "ts", StringComparison.OrdinalIgnoreCase))
-                {
-                    try
-                    {
-                        video.Timestamp = GetMpegTimestamp(video.Path);
+                return;
+            }
 
-                        _logger.LogDebug("Video has {Timestamp} timestamp", video.Timestamp);
-                    }
-                    catch (Exception ex)
-                    {
-                        _logger.LogError(ex, "Error extracting timestamp info from {Path}", video.Path);
-                        video.Timestamp = null;
-                    }
-                }
+            if (!string.Equals(video.Container, "mpeg2ts", StringComparison.OrdinalIgnoreCase) &&
+                !string.Equals(video.Container, "m2ts", StringComparison.OrdinalIgnoreCase) &&
+                !string.Equals(video.Container, "ts", StringComparison.OrdinalIgnoreCase))
```

Comment:
```suggestion
            if (!string.Equals(video.Container, "mpeg2ts", StringComparison.OrdinalIgnoreCase)
                && !string.Equals(video.Container, "m2ts", StringComparison.OrdinalIgnoreCase)
                && !string.Equals(video.Container, "ts", StringComparison.OrdinalIgnoreCase))
```

Reply:
Just for the future, is there a code style linter for stuff like this? E.g. in Kotlin, `detekt` wants you to put logical operators on the previous line. Here it seems to be the other way around.

Reply:
Not that I'm aware of